src/dev/amdgpu/amdgpu_nbio.cc

```diff
@@ -53,22 +53,38 @@ AMDGPUNbio::setGPUDevice(AMDGPUDevice *gpu_device)
 void
 AMDGPUNbio::readMMIO(PacketPtr pkt, Addr offset)
 {
+    // For Vega10 we rely on the golden values in an MMIO trace. Return
+    // immediately as to not clobber those values.
+    if (gpuDevice->getGfxVersion() == GfxVersion::gfx900) {
+        if (offset == AMDGPU_PCIE_DATA || offset == AMDGPU_PCIE_DATA2) {
+            return;
+        }
+    }
+
     switch (offset) {
-      // This is a PCIe status register. At some point during driver init
```

Comment:
any reason we can't update comment for new name?

Reply:
This comment was actually completely incorrect.  The driver will do an indirect write to update the value properly.  This was from a very early patch to get things working.  The new name is to match the name in the driver code